etc/kayobe/stackhpc-overcloud-host-images.yml

```diff
@@ -0,0 +1,28 @@
+---
+##############################
+# Overcloud host image sources
+
+# WIP - currently unused
```

Comment:
Maybe remove this bit if it's not being used yet.

Reply:
I think removing it would be best for now, I'll re add it once it is in use

Reply:
Done